Modules/Devices/src/Devices.Application/IntegrationEvents/Outgoing/IdentityStatusChangedIntegrationEvent.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus.Events;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+
+namespace Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
+public class IdentityStatusChangedIntegrationEvent : IntegrationEvent
+{
+    public IdentityAddress IdentityAddress { get; set; }
```

Comment:
Not really that important, but we typically place public properties after the constructor